### ISSUE TITLE
Pod Install Issues

### ISSUE THREAD

**Issue by jketch836** [GEOSwift/GEOSwift#61]
Hey there I am trying to install GEOSwift and I am getting this error when it is installing the geos pod:

Installing GEOSwift (2.1.0)
Installing geos (3.5.0)
[!] /bin/bash -c 
set -e
type -P autoconf &>/dev/null || alias autoconf 'xcrun autoconf'
type -P autoheader &>/dev/null || alias autoheader 'xcrun autoheader'
type -P aclocal &>/dev/null || alias aclocal 'xcrun aclocal'
type -P automake &>/dev/null || alias automake 'xcrun automake'
type -P glibtool &>/dev/null || alias glibtool 'xcrun glibtool'
type -P glibtoolize &>/dev/null || alias glibtoolize 'xcrun glibtoolize'

sh autogen.sh
./configure
./tools/svn_repo_revision.sh

sed -i "" "s/\/\* #undef HAVE_INT64_T_64 \*\//#define HAVE_INT64_T_64 1/" include/geos/platform.h
sed -i "" "s/#define HAVE_LONG_INT_64 1/\/\* #undef HAVE_LONG_INT_64 \*\//" include/geos/platform.h

cat <<EOT >> include/geos/platform.h
  #undef ISNAN
  #define ISNAN(x) (std::isnan(x))
EOT

patch capi/geos_c.h <<EOF
  149c149
  < #include <geos/export.h>
  ---
  > #define GEOS_DLL
EOF

/bin/bash: line 2: alias: autoconf: not found
/bin/bash: line 2: alias: xcrun autoconf: not found

I have installed autoconf, automake, and libtool, they are in my home directory. I have installed using the coocapods app as well as in terminal. I am running it on Xcode 9.0.1 and running the latest version of cocoapods (not the beta). Any thoughts?

**Comment by vfn** [comment]
@jketch836 

Please have a look at the [readme](https://github.com/GEOSwift/GEOSwift#cocoapods) and on some of these closed issues: https://github.com/GEOSwift/GEOSwift/issues?utf8=%E2%9C%93&q=is%3Aissue%20%2Fbin%2Fbash%3A%20line%202%3A%20alias%3A%20autoconf%3A%20not%20found

**Comment by jketch836** [comment]
Actually I have also read the closed issues. Those didn't seem to help.

**Comment by vfn** [comment]
Cool! Could you create a sample project, and I’ll have a look at it?

Thanks!

**Comment by jketch836** [comment]
I can download the GEOSwift master branch and install the podfile and it won't work. I think it might be a file path issue. Terminal acknowledges they are there in the right place (the home directory) according to stack overflow. Do you know of any issues with APFS?

**Comment by vfn** [comment]
@johncrenshaw, I don't know of any issues with APFS. There shouldn't be any. if the tools are installed and accessible via terminal, I can't se why the build script cannot find them.

Could you go inside the `GEOSwift` folder via terminal and run the following command:
```bash
which autoconf & which automake & which libtool
```

and then
```bash
xcrun autoconf --version
```

Please paste the line where you run the command, and the output

Thanks!

**Comment by jketch836** [comment]
which autoconf
/usr/local/bin/autoconf

which automake
/usr/local/bin/automake

which libtool
/usr/local/bin/libtool


xcrun autoconf --version
2017-10-19 15:35:49.730 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/XcodeColors.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.730 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/RealmPlugin.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.731 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/RealmBrowser.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.731 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/OMColorSense.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.731 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/IconMaker.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.731 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/HOStringSense.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.732 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Crayons.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.732 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/ColorSenseRainbow.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.732 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/CocoaPodUI.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.733 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/CocoaPods.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.733 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Backlight.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.734 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/AMMethod2Implement.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.734 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/AllTargets.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.734 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/Alcatraz.xcplugin' not present in DVTPlugInCompatibilityUUIDs
2017-10-19 15:35:49.735 xcodebuild[1809:44314] [MT] PluginLoading: Required plug-in compatibility UUID DF11C142-1584-4A99-87AC-1925D5F5652A for plug-in at path '~/Library/Application Support/Developer/Shared/Xcode/Plug-ins/ACCodeSnippetRepository.xcplugin' not present in DVTPlugInCompatibilityUUIDs

autoconf (GNU Autoconf) 2.69
Copyright (C) 2012 Free Software Foundation, Inc.
License GPLv3+/Autoconf: GNU GPL version 3 or later
<http://gnu.org/licenses/gpl.html>, <http://gnu.org/licenses/exceptions.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

Written by David J. MacKenzie and Akim Demaille.

**Comment by vfn** [comment]
@jketch836 It looks like you are all set...unfortunately I don't think I know how to help you on this issue.

**Comment by jketch836** [comment]
So I found out the issue. First I tried the Cocoapods app, I tried running Install, Install (verbose), and Update (verbose). Those did not work. I still got the error that the pods did not install correctly. I then opened terminal and changed the directory to my project folder and ran the commands that you gave me above in terminal, then tried pod update. It worked. GEOSwift and the geos were installed. Apparently it needed to know where autoconf, automake and libtool was before it installed properly.
I know this might not be a big issue, but you might want to add to the NOTE under Installation that the computer needs to know the path where autoconf, automake and libtool are located before GEOSwift can be installed. This issue drove me nuts.
